gap/semigroups/semitrans.gi

```diff
@@ -328,17 +328,30 @@ SEMIGROUPS.GraphOfRightActionOnPairs := function(gens, n, stop_on_isolated_pair)
 end;
 
 # FIXME can probably do better than this
+# WW has improved this
 
```

Comment:
If this really improves the performance, then you could remove the two comments before this method. In particular, please remove the FIXME.